src/Memed/Soluti/Auth/Credentials.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Memed\Soluti\Auth;
+
+/**
+ * This class is a simple DTO to handle user's credentials.
+ */
+class Credentials implements AuthStrategy
+{
+    /**
+     * @var Client
+     */
+    protected $client;
+
+    /**
+     * @var string
+     */
+    protected $username;
+
+    /**
+     * @var string
+     */
+    protected $password;
+
+    /**
+     * @var int
+     */
+    protected $ttl;
+
+    /**
+     * Constructor.
+     */
+    public function __construct(
+        Client $client,
+        string $username,
+        string $password,
+        int $ttl
+    ) {
+        $this->client = $client;
+        $this->username = $username;
+        $this->password = $password;
+        $this->ttl = $ttl;
+    }
+
+    /**
+     * Retrieves client.
+     * @return [type] [description]
```

Comment:
**nip**: I think it was added by docblock autocomplete.

Reply:
Nice, I will remove it.